test/Concurrency/global_actor_inference.swift

```diff
@@ -94,6 +94,17 @@ func testNotAllInP1(nap1: NotAllInP1) { // expected-note{{add '@SomeGlobalActor'
   nap1.other() // okay
 }
 
+// Make sure we don't infer 'nonisolated' for stored properties.
+@MainActor
+protocol Interface {
+  nonisolated var baz: Int { get } // expected-note{{'baz' declared here}}
+}
+
+@MainActor
+class Object: Interface {
+  var baz: Int = 42 // expected-warning{{property 'baz' isolated to global actor 'MainActor' can not satisfy corresponding requirement from protocol 'Interface'}}
```

Comment:
What's the fix here, explicitly annotating it with @MainActor? If so, can we get a fixit?

Reply:
The fix here would have to be to turn it into a non-isolated computed property.

Reply:
ah, makes sense 👍 Could also be a fixit i guess